maturity-level-one/src/Controllers/PlayersController.cs

```diff
@@ -33,24 +33,27 @@ public PlayersController(IWorldCupRepository worldCupRepository)
         [SwaggerOperation("get-all-players")]
         [SwaggerResponse(200, "OK")]
         [SwaggerResponse(500, "API is not available")]
-        public async Task<IActionResult> GetPlayers([FromQuery(Name = "top-players-only")]bool topPlayersOnly)
+        public async Task<IActionResult> GetPlayersAsync([FromQuery(Name = "top-players-only")]bool topPlayersOnly)
         {
-            var players = await _worldCupRepository.GetAllPlayers(topPlayersOnly);
+            var players = await _worldCupRepository.GetAllPlayersAsync(topPlayersOnly);
             var results = Mapper.Map<IEnumerable<PlayerDto>>(players);
             return Ok(results);
         }
 
         [HttpPost("{id}/vote")]
         [SwaggerOperation("vote-as-best-player")]
-        [SwaggerResponse(204, "No Content")]
+        [SwaggerResponse(202, "Vote accepted")]
```

Comment:
I'll leave this one up to @MassimoC to decide but I would tend to use 204 here

Reply:
Because we already have other operations with 204 I would prefer to implement this with 202 + location header